src/usb/uf2/ghostfat.c

```diff
@@ -518,5 +565,6 @@ int write_block (uint32_t block_no, uint8_t *data, WriteState *state)
     }
   }
 
-  return 512;
+  STATIC_ASSERT(BPB_SECTOR_SIZE == 512); // if sector size changes, may need to re-validate this code
```

Comment:
despite of whatever Microsoft says, sector never CHANGES :laughing: :laughing:

Reply:
For FAT16, you are right.

Reply:
nobody will change this value even with FAT32, probably only existed in other place than embedded :)